http_server/queries/users/user_delete.php

```diff
@@ -1,91 +1,401 @@
 <?php
 
```

Comment:
Dang, this must have taken forever :+1:

Reply:
Copy/paste is the best thing ever known to man.  Still, it took about 20 minutes just for this file. :expressionless: